### ISSUE TITLE
@solana-developers/helpers getKeypairFromEnvironment import not working

### ISSUE THREAD

**Issue by Jams455** [solana-developers/helpers#54]
Hello! 

I am following the solana web3.js tutorials and am on [this](https://solana.com/developers/courses/intro-to-solana/intro-to-cryptography#loading-an-existing-keypair) step. To recreate my issue as simply as possible I have run the following commands:

````
mkdir deleteme
cd deleteme
npm init -y
npm i @solana-developers/helpers
npm i esrun
npx esrun runme.ts
````

runme.ts:
````
import { getKeypairFromEnvironment } from "@solana-developers/helpers";
import 'dotenv/config';
console.log(getKeypairFromEnvironment("SECRET_KEY"));`
````
where the .env file in the same directory contains the secret key variable.
The error I get is:
```
node:internal/modules/run_main:129
    triggerUncaughtException(
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module 'C:\Users\******\Documents\B - Dev\deleteme\node_modules\.bin\node_modules\@solana-developers\helpers\dist\esm\index.js' imported from C:\Users\******\Documents\B - Dev\deleteme\node_modules\.bin\esrun-1726001953025.tmp.mjs
Did you mean to import "../@solana-developers/helpers/dist/esm/index.js"?
    at finalizeResolution (node:internal/modules/esm/resolve:265:11)
    at moduleResolve (node:internal/modules/esm/resolve:933:10)
    at defaultResolve (node:internal/modules/esm/resolve:1169:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:540:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:509:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:239:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:96:40)
    at link (node:internal/modules/esm/module_job:95:36) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///C:/Users/******/Documents/B%20-%20Dev/deleteme/node_modules/.bin/node_modules/@solana-developers/helpers/dist/esm/index.js'
}

Node.js v20.17.0
```

I am not sure what is wrong. I have tried reinstalling @solana-developers/helpers including previous versions as suggested [here](https://github.com/solana-developers/helpers/issues/42#issuecomment-2303582640) to no avail. I have also manually checked all package.json files and as far as I can tell all exports seem to be okay. 

I am also not sure why the node_modules/.bin/ directory is being looked in as there are no folders in there. 

Any help would be much appreciated and sorry to raise import issues yet again - I have read through all of the previous ones. Apologies if this is an issue with node/me being stupid and not solana-development.

**Comment by Jams455** [comment]
Update - after running:
```
npm cache clean --force
rm -rf node_modules package-lock.json
npm install
```

the error message no longer seemed to suggest it was looking in any /.bin/ directory which seemed positive so clearly this bit was an issue on my end, however imports were still not working. 

I then rolled back to version 2.5.2 of @solana-developers/helpers and this fixed the issue completely, allowing getKeypairFromEnvironment to be imported successfully. So I wonder if [this](https://github.com/solana-developers/helpers/issues/42) bug has repeated itself in @solana-developers/helpers@latest.

**Comment by heyAyushh** [comment]
can you also share your package.json @0xCipherCoder ?

**Comment by heyAyushh** [comment]
I only wanted to know if you have "type": "module" in your package.json

and I think yes its the error in ESM distribution probably due to postbuild script

**Comment by heyAyushh** [comment]
This has been fixed with v2.5.6
